src/glue/jobs/s3_to_json.py

```diff
@@ -398,7 +409,11 @@ def write_file_to_json_dataset(
             os.remove(output_path)
     return uploaded_files
 
-def get_part_path(metadata: dict, part_number: int, dataset_identifier: str, touch: bool):
```

Comment:
Did you want to mention `partition` somewhere here either in the function name or the description?

Reply:
No, these "parts" aren't actually related to the `cohort` partition. Although it is true that we are partitioning the data by writing it in parts, that terminology has the potential of becoming confounded with the hive-style partitions we get when writing the data to S3.

Here's a diagram of how this looks in S3:
- dataset_datatype
  * cohort=adults_v1 <- [ Hive partition ]
    - part0.snappy.parquet <- [ part "partition" ]
    - part1.snappy.parquet
    - ...
  * cohort=pediatric_v1
    - part0.snappy.parquet
    - part1.snappy.parquet
    - ...